ci/azure-cross-compile.yml

```diff
@@ -37,7 +53,7 @@ jobs:
     - script: rustup target add $(target)
       displayName: "Add target"
 
-    - script: cargo check --target $(target)
+    - script: cargo build --target $(target)
```

Comment:
Why change `cargo check` to `cargo build`?  Won't that just slow things down for little benefit?

Reply:
That was in the original pr, see https://github.com/tokio-rs/mio/pull/952/files#diff-424edf9074d0455636ef5bf26a0c9cd4R56. @carllerche any reason why you changed it?

Reply:
I'm fine either way. So up to y'all.

Reply:
I don't remember why I changed it

Reply:
I'll change it back, because I do think its faster.